### ISSUE TITLE
The pretrained model ?

### ISSUE THREAD

**Issue by Leesoon1984** [xuyu0010/PATAN#2]
Excellent Job! Can you provide the link to the pretrained model ?

**Comment by xuyu0010** [comment]
Hi Lee Soon,
Thank you for your interest! I have just found out that the original link to the pretrained model is no longer available. I will try to upload to a cloud server and share the new link as soon as possible.

**Comment by xuyu0010** [comment]
> Excellent Job! Can you provide the link to the pretrained model?

Hi Lee Soon,
Thank you for your interest! I have just found out that the original link to the pretrained model is no longer available. I will try to upload to a cloud server and share the new link as soon as possible.

**Comment by Leesoon1984** [comment]
Thank you very much for your reply.

**Comment by Leesoon1984** [comment]
> > Excellent Job! Can you provide the link to the pretrained model?
> 
> Hi Lee Soon, Thank you for your interest! I have just found out that the original link to the pretrained model is no longer available. I will try to upload to a cloud server and share the new link as soon as possible.

Thank you very much for your reply.

**Comment by Leesoon1984** [comment]
> 
In the paper, the initial learning rate is set to 0.005 and is divided by 10 for two times during the training process.
How does the learning rate change in the code? 

`parser.add_argument('--lr-steps', type=list, default=[int(1e4*x) for x in [1.5,3,4]]) `

It seems that different data should be set differently?

**Comment by xuyu0010** [comment]
> > 
> 
> In the paper, the initial learning rate is set to 0.005 and is divided by 10 for two times during the training process. How does the learning rate change in the code?
> 
> `parser.add_argument('--lr-steps', type=list, default=[int(1e4*x) for x in [1.5,3,4]]) `
> 
> It seems that different data should be set differently?

Yes, the settings here are a bit out of date and we will update this to a dynamic version as soon as possible. We are currently a bit busy with NeurIPS so please understand that there may be a short delay in the update. Closing this issue as the pretrained model is now available.
